src/main/twirl/gitbucket/core/releases/release.scala.html

```diff
@@ -48,8 +48,8 @@ <h3>
                 <span class="label label-default">@helpers.readableSize(Some(asset.size))</span>
               </li>
             }
-            <li><a href="@helpers.url(repository)/archive/@{helpers.encodeRefName(release.tag)}.zip"><i class="octicon octicon-file-zip"></i>Source code (zip)</a></li>
-            <li><a href="@helpers.url(repository)/archive/@{helpers.encodeRefName(release.tag)}.tar.gz"><i class="octicon octicon-file-zip"></i>Source code (tar.gz)</a></li>
+            <li><a href="@helpers.url(repository)/archive/@{helpers.encodeRefName(release.tag)}?format=zip"><i class="octicon octicon-file-zip"></i>Source code (zip)</a></li>
+            <li><a href="@helpers.url(repository)/archive/@{helpers.encodeRefName(release.tag)}?format=tar.gz"><i class="octicon octicon-file-zip"></i>Source code (tar.gz)</a></li>
```

Comment:
Could you also update `zipball_url` and `tarball_url` added in https://github.com/gitbucket/gitbucket/pull/2544? 🙇